virtual_rainforest/models/abiotic/abiotic_model.py

```diff
@@ -31,7 +31,7 @@ class as a child of the :class:`~virtual_rainforest.core.base_model.BaseModel` c
 from virtual_rainforest.core.data import Data
 from virtual_rainforest.core.logger import LOGGER
 from virtual_rainforest.core.utils import set_layer_roles
-from virtual_rainforest.models.abiotic import wind
+from virtual_rainforest.models.abiotic import energy_balance, wind
 from virtual_rainforest.models.abiotic.constants import AbioticConsts
```

Comment:
I think this might be unnecessary here as the constants are imported via load_config?